src/selfie_avoiding_obstacles/src/road_obstacle_detector.cpp

```diff
@@ -40,25 +59,38 @@ void Road_obstacle_detector::obstacle_callback(const selfie_msgs::PolygonArray &
   case CLEAR:
     filter_boxes(msg);
```

Comment:
Nawet jak jest czysto to dla pewności wysyłałbym cały czas right_lane offset tak jak to robił dotychczasowy node